### PR TITLE
Fab 3982

### PR DESCRIPTION
**Pull request by akprasad-cs**
# Checklist:
Please check you fulfill ALL of the relevant checkboxes
- [x] Notified docs of any potential USER-facing changes
- [x] Added short description of the change - with relevant motivation and context. 
- [x] Branch has the ticket number in its name (along with a ticket summary)
- [x] Commented the code, particularly in hard-to-understand areas
- [x] Added tests that prove my fix is effective or that my feature works
- [x] Ran `npm test` and it passes
- [x] Changes generate no new warnings
image

### CODE REVIEW

File: bin/cortex-agents.js [hunk 1]
```diff
@@ -68,9 +68,9 @@ program
     .option('--project [project]', 'The project to use')
     .option('--json', 'Output results using JSON')
     .option('--query [query]', 'A JMESPath query to use in filtering the response data. Ignored if output format is not JSON.')
-    .option('--skip [skip]', 'Move the result cursor to this position before returning results.')
-    .option('--limit [limit]', 'Limit the number of rows returned')
     .option('--filter [filter]', 'A Mongo style filter to use.')
+    .option('--limit [limit]', 'Limit number of records')
```
Comment (pkandarpa-cs) [inline bin/cortex-agents.js:72]: should we have a global constant as the default limit for all commands?

Comment (pkandarpa-cs) [reply bin/cortex-agents.js:72]: some options have `100` specified and some don't. need to make this consistent across ALL commands

File: src/client/catalog.js [hunk 2]
```diff
@@ -49,10 +49,9 @@ module.exports = class Catalog {
             .catch((err) => constructError(err));
     }
 
-    listSkills(projectId, token, query, filter, limit, skip, sort) {
+    listSkills(projectId, token, query, limit, skip, sort) {
```
Comment (pkandarpa-cs) [inline src/client/catalog.js:52]: why are we removing `filter` ?

File: bin/cortex-agents.js (hunk 1 above)
Comment (akprasad-cs) [reply bin/cortex-agents.js:72]: 100 rows seem to be a lot of data to view in a cli. Should we reduce this count?

Comment (mchutani-CS) [reply bin/cortex-agents.js]: I think a good pattern is to have 20 as global default, and can be overridden with --limit param.

**Review by clee-cs — APPROVED** [review]

File: bin/cortex-agents.js (hunk 1 above)
Comment (akprasad-cs) [reply bin/cortex-agents.js]: @yreddy-CS will restricting the default limit to 20, break the integration tests? Remember there were large number of connections being created in the tests and were being captured.

**Review by clee-cs — APPROVED** [review]

**Pull request merged**